tests/incremental/thinlto/cgu_keeps_identical_fn.rs

```diff
@@ -33,12 +33,12 @@
 mod foo {
 
     // Trivial functions like this one are imported very reliably by ThinLTO.
-    #[cfg(any(cfail1, cfail4))]
+    #[cfg(any(cfail1, cfail4))] //~ WARN unexpected `cfg` condition name: `cfail4`
     pub fn inlined_fn() -> u32 {
         1234
     }
 
-    #[cfg(not(any(cfail1, cfail4)))]
+    #[cfg(not(any(cfail1, cfail4)))] //~ WARN unexpected `cfg` condition name: `cfail4`
```

Comment:
This caught a mistake in the test, the revision `cfail4` did exist in the past, but no longer exist.
I'll remove it, but I just wanted it to be visible in the PR first.

Reply:
Ah! Good point